src/nls/fi/strings.js

```diff
@@ -624,7 +624,25 @@ define({
     "CMD_TOGGLE_RECENT_PROJECTS"                : "Viimeisimmät projektit",
 
     // extensions/default/WebPlatformDocs
-    "DOCS_MORE_LINK"                            : "Lue lisää"
+    "DOCS_MORE_LINK"                            : "Lue lisää",
+
+    //extensions/default/CodeFolding
+    "COLLAPSE_ALL"                  : "Pienennä kaikki",
```

Comment:
I failed to notice that brackets didn't have submenus; i guess this is the way for now.